scripts/ci.publish.js

```diff
@@ -24,19 +24,19 @@ function validatePrerequisites() {
     throw new Error(`Cannot publish: lerna not installed!`);
   }
 
-  const lernaVersion = exec.execSyncRead('lerna --version');
-  if (!lernaVersion.startsWith('2.')) {
-    throw new Error(`Cannot publish: lerna version isn't 2.x.x (actual version is ${lernaVersion})`);
-  }
+  // const lernaVersion = exec.execSyncRead('lerna --version');
+  // if (!lernaVersion.startsWith('2.')) {
+  //   throw new Error(`Cannot publish: lerna version isn't 2.x.x (actual version is ${lernaVersion})`);
+  // }
```

Comment:
delete me! ❤️

Reply:
On 2nd thought, why not keep this validation as-is?

Reply:
I just have expressed my confusion about why can't we move to Lerna 3, by using `npx lerna ...` local CLI.

Reply:
A bit more complicated than that, but looks like its working now